frontend/packages/portal-frontend/src/compound/components/DoseResponseCurve.tsx

```diff
@@ -79,6 +80,7 @@ export class DoseResponseCurve extends React.PureComponent<DoseResponseCurveProp
     measurements: defaultPoints,
     curves: defaultParams,
     yUnits: "DEFAULT NOT FOUND",
+    xUnits: undefined,
```

Comment:
I don't know what this is used for, but it looks like setting it to "DEFAULT NOT FOUND" would be consistent with what's being done for yUnits.